external/js/cothority/lib/byzcoin/Proof.js

```diff
@@ -26,26 +24,22 @@ class Proof {
    * stored, false if it is a proof of absence.
    */
   matches() {
-    return this._leaf !== undefined;
+    // TODO write a proper way to do match
```

Comment:
Now or later?

Reply:
in a future PR, because java/js never had a proper `match` implementation